scss/components/_icon.scss

```diff
@@ -1,3 +1,5 @@
 .civicase__icon {
-  transform: translateY(14%);
+  // rotate(0.03deg) is applied to retain proper shape of icons in low resolution
+  // https://github.com/google/material-design-icons/issues/648#issuecomment-335735429
+  transform: translateY(14%) rotate(0.03deg);
```

Comment:
What happens to big resolution icons? This is supposed to be a solution for something else, not necessarily small icons.

Can you check this does not affect big icons?